core/game_object.lua

```diff
@@ -805,22 +809,16 @@ function loadAPIs()
             SMODS.process_loc_text(G.localization.descriptions.Other, 'undiscovered_' .. string.lower(self.key),
                 self.loc_txt, 'undiscovered')
         end,
-        generate_colours = function(self, base_colour, alternate_colour)
-            if not self.colour_shifter then return HEX("000000") end
+        generate_colours = function(self, base_colours)
```

Comment:
Concerned about adding this- I don't want to add things to SMODS that are not generally useful utilities, this seems pretty specific.